drgn/helpers/linux/mm.py

```diff
@@ -124,6 +124,21 @@
 )
 
 
+def _get_page_flags(page: Object) -> int:
```

Comment:
This should probably be a public helper, so drop the leading underscore and add it to `__all__`.